pages/orders.js

```diff
@@ -0,0 +1,135 @@
+import Layout from '../components/Layout'
+import useSWR from 'swr'
+import cookie from 'js-cookie';
+
+const fetcher = (url,token) => fetch(url,{headers: {'Content-Type': "application/json", 'Authorization': token}}).then((res) => res.json())
+
+const token = cookie.get("firebaseToken")
+
+class PackingOrders extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+          orders: props.data      
+        }
+    }
+
+    // Permanently deletes order
+    deleteOrder(orderId) {
+      fetch("/api/orders/DeleteOrder", {
+        method: "POST",
+        body: JSON.stringify({
+          orderId: orderId,
+        }),
+        headers: { "Content-Type": "application/json", Authorization: token },
+      }).then(() => {
+        for (var i = 0; i < this.state.orders.length; i++) {
+          if (this.state.orders[i].id == orderId) {
+            this.state.orders.splice(i, 1) // remove order from array
+            this.setState({orders: this.state.orders})
+            break
+          }
+        }
+      });
+    }
+
+    displayOrderRow(order) {
+      return (
+      <tr className="h-10" key={order.id}>
+      <td className="w-auto"><a href={order.url}>{order.name}</a></td>
+      <td className="w-auto">{order.numBags}</td>
+      <td className="w-auto">{order.deliveryDate}</td>
+      <td className="w-auto">{order.deliveryWindow}</td>
+      <td className="w-auto">{order.status}</td>
+      <td className="w-auto">
+      <button className="font-bold text-xl" onClick={() => this.deleteOrder(order.id)}>
+      <svg className="w-5 h-5" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M15 12H9m12 0a9 9 0 11-18 0 9 9 0 0118 0z"></path></svg>
```

Comment:
I used a trash can SVG for checkout page if you wanted to use that instead

rplace `<svg...>` with `<img src="/images/trash-can.svg"></img>` and then just add the `w-5 h-5` to the button class

Reply:
Changed to the trash can :)